examples/sample-server/src/node/services/PersonService.ts

```diff
@@ -11,15 +11,8 @@ interface PersonRequest {
 
 class PersonService {
 
-  async create(request:PersonRequest):Promise<PersonResponse> {
-    const id = uuidv4();
-    const model = Person.build({ id });
-    model.first_name = request.first_name;
-    model.last_name = request.last_name;
-    model.age = request.age;
```

Comment:
It is more secure if we explicitly define the properties on the model, although TypeScript is typed, it does not ensure that the object received has only the defined properties, that may lead to property injection.